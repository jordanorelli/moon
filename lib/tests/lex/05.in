
errant_name

no_value:

key: var_name

key: "string"
