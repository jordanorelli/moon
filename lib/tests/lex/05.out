{t_name errant_name}
{t_name no_value}
{t_object_separator :}
{t_name key}
{t_object_separator :}
{t_name var_name}
{t_name key}
{t_object_separator :}
{t_string string}
